static/js/vue-cdr-access/src/components/filterTags.vue

```diff
@@ -106,7 +108,8 @@ export default {
             };
 
             // Return non multi-value tags
-            if (type !== 'format' && type !== 'collection') {
+            const MULTI_VALUED_FIELDS = ['format', 'collection', 'creatorContributor'];
```

Comment:
Would probably make sense to declare this as a module level constant rather than in this function, maybe up next to `TYPE`